Loop/View Controllers/SettingsTableViewController.swift

```diff
@@ -16,6 +16,7 @@ import LoopTestingKit
 import LoopUI
 
 final class SettingsTableViewController: UITableViewController, IdentifiableClass {
+    private let temporaryNewSettingsViewTesting = true
```

Comment:
This flag is the "switch" that hides the new settings screen inside this screen, replacing the "Alert Permissions" button (and so "Alert Permissions" would be accessed via that new screen)